spec/models/category_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  it { is_expected.to validate_presence_of(:title) }
```

Comment:
It is not necessary.

Reply:
@denys281  I don't want the ability to create categories without titles, so I prefer to leave that validation, and also this test

Reply:
As you wish) but I prefer not to write specs for relations and validations)